tests/gnu/test_arch.rs

```diff
@@ -6,28 +6,35 @@
 // For a copy, see the LICENSE file.
 //
 
-use util::*;
+use std::process::Command;
+use assert_cmd::prelude::*;
 
 #[test]
 #[cfg(target_arch = "x86_64")]
 fn test_x86_64() {
-    new_ucmd!()
-        .succeeds()
-        .stdout_only("x86_64\n");
+    new_cmd!()
+        .assert()
+        .success()
+        .stdout("x86_64\n")
```

Comment:
We're considering trimming whitespace for you.

Could you post your thoughts on https://github.com/assert-rs/assert_cmd/issues/19